spec/lib/video_info_spec.rb

```diff
@@ -25,6 +25,29 @@
     end
   end
 
+  describe ".disable_providers" do
+    let(:youtube_url) { 'https://www.youtube.com/watch?v=mZqGqE0D0n4' }
+    let(:vimeo_url) { 'http://vimeo.com/86701482' }
+
+    it "does not attempt to use a provider marked as disabled" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.